modules/react/text/lib/LabelText.tsx

```diff
@@ -1,18 +1,56 @@
 import * as React from 'react';
 import {Property} from 'csstype';
 import {createComponent} from '@workday/canvas-kit-react/common';
-import {Text, TextProps} from './Text';
-import {inputColors} from '@workday/canvas-kit-react/tokens';
+import {base, system} from '@workday/canvas-tokens-web';
+import {CSProps, createStencil} from '@workday/canvas-kit-styling';
+import {CommonStyleProps, mergeStyles} from '@workday/canvas-kit-react/layout';
 
-export interface TypeLabelProps extends TextProps {
+export interface TypeLabelProps extends CSProps, CommonStyleProps {
   cursor?: Property.Cursor;
   disabled?: boolean;
+  /**
+   * Type variant token names: `error`, `hint` or `inverse`.
+   *
+   * ```tsx
+   * <LabelText variant="error">Error text</LabelText>
+   * ```
+   */
+  variant?: 'error' | 'hint' | 'inverse';
 }
 
+const labelTextStencil = createStencil({
+  // @ts-ignore
```

Comment:
instead of ignoring all of base, can you just ignore the line below?

Reply:
Let's do that

Reply:
oh no, this is not possible to do. It shows ts error in `base` if replaced